x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/support/SecurityIndexManagerTests.java

```diff
@@ -408,43 +404,6 @@ public void testCanUpdateIndexMappings() {
         assertThat(putMappingRequestCount, equalTo(1));
     }
 
-    /**
```

Comment:
When there are nodes in the cluster on a descriptor with a mapping version less than the current node, the expected behaviour should (?) be that the mappings are not updated by the current node, instead that's left to be handled by the node with the min version and the query will run anyway. 

It could also be argued that if the min version is for example `2`, what's in the index is `1`, while the descriptor of the current node is version `3`, the query should fail. But I'm not sure if that's really necessary. It depends on how strict we want to to be with `prepareIndexIfNeededThenExecute`. The prepare action would always fail on the node with `3` if we want to be really strict, since we can't apply the "latest" mapping as long as we're in a mixed cluster.